pkgs/by-name/ka/kakasi/avoid-an-implicit-declaration-of-exit.patch

```diff
@@ -0,0 +1,26 @@
+From 9d66aa1f2797f70701bd601c4565043bc4333ebb Mon Sep 17 00:00:00 2001
```

Comment:
I believe the header should be removed, especially the part with identifying information

Maybe keep the launchpad link next to the path in the `package.nix`?

Reply:
You mean, turn it into a diff? Why?

Reply:
To avoid adding irrelevant information to the nixpkgs repo.  What benefit is there of keeping?

```
configure | 2 +-
1 file changed, 1 insertion(+), 1 deletion(-)
``` 

There are a lot of projects referred to by the nix monorepo, each with vast quantities of irrelevant information about them that could be included, but isn't to avoid pointless bloat, slower evals, slower clones, etc.

This is of course a minor example of that, but I still see so no reason halve the size of this file, so I think it should be done

Reply:
Okay, it is a fairly reasonable request.